src/switch/switch.component.ts

```diff
@@ -120,19 +121,5 @@ export class SwitchComponent extends CheckboxComponent implements OnInit {
 	 * Builds variant classes and appends them to the switch and label elements.
 	 * @memberof SwitchComponent
 	 */
-	ngOnInit() {
-		/* TODO: remove and extend in neutrino
-		// Build variant classes
-		const labelClass = `toggle-label${this.size !== "md" ? `--${this.size}` : ""}`;
-		const buttonClass = `toggle${this.size !== "md" ? `--${this.size}` : ""}`;
-
-		// Get elements
-		const labelEl = this.elementRef.nativeElement.querySelector("label");
-		const buttonEl = this.elementRef.nativeElement.querySelector("button");
-
-		// Add classes to elements
-		this.renderer.addClass(labelEl, labelClass);
-		this.renderer.addClass(buttonEl, buttonClass);
-		*/
-	}
+	ngOnInit() { }
```

Comment:
If we're removing this it makes sense we remove it completely. `implements` part too.
```suggestion
```